srcpkgs/dpp/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'dpp'
+pkgname=dpp
+version=10.0.23
+revision=1
+build_style=cmake
+configure_args="-DDPP_USE_EXTERNAL_JSON=ON"
+cmake_builddir=build
+hostmakedepends="pkg-config"
+makedepends="libsodium-devel opus-devel zlib-devel openssl-devel json-c++"
+short_desc="C++ Discord API Library for Bots"
+maintainer="Lunar Starstrum <lunar@forsaken-borders.net>"
+license=Apache-2.0
+homepage="https://dpp.dev"
+distfiles="https://github.com/brainboxdotcc/DPP/archive/refs/tags/v${version}.tar.gz"
+checksum=8f9db61c3586a492ada378235300c509e3bc2fc090cef32de0a8241741038df0
+
+dpp-devel_package() {
+	depends="${makedepends} ${sourcepkg}-${version}_${revision}"
+	short_desc+=" - development files"
+	pkg_install() {
+		# If the path already exists, remove it to prevent conflicts
+		filepath="${DESTDIR}/usr/lib/dpp-${version%.*}"
```

Comment:
how does this create conflicts?

Reply:
This was on the previous PR's. I was unsure if this was still needed. Am I safe to remove it?

Reply:
It's done here https://github.com/void-linux/void-packages/pull/38601/files#diff-1f1bf3035f39fc9c91e206b7219fd65d0544cdc55d7330fddf5f0e5b0d5e2419R25 though I don't know why. My guess was to remove previous files if the package was being reinstalled.

Reply:
I don't know why the package would be getting reinstalled in the `masterdir/`, install phase can only run once. Does something actually break when you remove it?

Reply:
It... doesn't appear to. I'll go ahead and remove it then